cwizard2011/maintenace-tracker-app
# #157703616 Balance colors and button size

#### What does this PR do?
- Change background image
- Change color of the navigation bar
- Change icons on the landing page
- Change card size and edit card content on the dashboard page
- Change button style, card size, and status icon on admin dashboard page

#### How should this be manually tested?
clone this repo, navigate to the ```index.html```, ```dashboard.html```, and ```admindashboard.html``` files. check the responsiveness with chrome developer tool. You can also check on any device by visiting this link https://cwizard2011.github.io/maintenace-tracker-app/UI/

#### What are the relevant pivotal tracker stories?
#157703616
